Model/PaymentGateway.php

```diff
@@ -7,7 +7,7 @@
  * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
  */
 
-namespace Wirecard\Oxid\Extend\Model;
+namespace Wirecard\Oxid\Model;
```

Comment:
Why? This still extends a model (`OxidEsales\EshopCommunity\Core\Model\BaseModel`).

Reply:
Ah, yes, the extension was added later again.